tests/collections/options.test.ts

```diff
@@ -83,9 +83,9 @@ describe(`Options tests`, async () => {
 
     async function dropCollections(isAstra: boolean, astraMongoose: mongoose.Mongoose, jsonAPIMongoose: mongoose.Mongoose, collectionName: string) {
```

Comment:
Maybe we move `({ Product, astraMongoose, jsonAPIMongoose } = await createClientsAndModels(isAstra));` to a `beforeEach()` and `await dropCollections(isAstra, astraMongoose, jsonAPIMongoose, 'products');` to an `afterEach()`? As written, this code is a bit repetitive, and using `afterEach()` for test cleanup is a cleaner pattern than wrapping the whole test in a `try/catch`.

Reply:
Agreed, fixed it.